app/templates/search/_briefs_results.html

```diff
@@ -18,7 +18,7 @@ <h2 class="search-result-title">
             {{ brief.lotName }}
         </li>
         <li class="search-result-metadata-item">
-            {{ brief.frameworkName }}
+            {{ brief.specialistRole }}
```

Comment:
I had a look and I think it's fine to show nothing for other lots:
![screen shot 2017-06-01 at 17 54 59](https://cloud.githubusercontent.com/assets/6525554/26691011/815f6726-46f3-11e7-8c4e-40ec7968f12f.png)